crates/storage/provider/src/providers/static_file/manager.rs

```
@@ -1416,6 +1416,13 @@ impl<N: NodePrimitives<SignedTx: Value + SignedTransaction, Receipt: Value>> Rec
             |_| true,
         )
     }
+
+    fn receipts_by_block_range(
+        &self,
+        _block_range: RangeInclusive<BlockNumber>,
+    ) -> ProviderResult<Vec<Vec<Self::Receipt>>> {
+        unreachable!()
```

let's also return an error here, just to be consistent